wordpress-mobile/WordPress-iOS
# Add Calendar Picker + new Publish Settings screen

Ref #12686

This addresses the first items from the master issue:
- [x] Add feature flag. (`postScheduling`)
- [x] Add new 'Publish' flow to select publish date and time.

<div align="center">
<img width="486" alt="Napkin 2 10-21-19, 10 27 23 PM" src="https://user-images.githubusercontent.com/3250/67259243-f5825b00-f451-11e9-930f-e0f789c4fcc7.png">
</div>

To test:
- Navigate to the `Published` setting on a blog post.
- Tap on the Published cell
- Review new Publish settings screen

A few things to point out:
- Adds a new dependency on [JTAppleCalendar](https://github.com/patchthecode/JTAppleCalendar)
- The new `HalfScreenPresentationController` might be useful in other parts of the app and could possibly be moved to WordpressUI alongside its `FancyAlertPresentationController` parent

There are a few known issues I am hoping to address shortly:
- Rotation to landscape does not work well with the new `HalfScreenPresentationController`
- The bottom sheet should probably be in a popover but that may not be critical for shipping.

Please feel free to leave any feedback or ask any questions about the design. I tried to stick with in-code view layout/setup because most of the global styling stuff seemed much more easily accessible that way.
